### ISSUE TITLE
Invalid method Nexcessnet_Turpentine_Model_Core_Session::validateFormKey

### ISSUE THREAD

**Issue by viansolutions-com** [nexcess/magento-turpentine#1107]
After applying some Magento security patches, we are getting this error while trying to login in Backoffice

Problem comes because Turpentine declares this rewrite

`            <core>
                <rewrite>
                    <session>Nexcessnet_Turpentine_Model_Core_Session</session>
                </rewrite>
            </core>
`

But, Nexcessnet_Turpentine_Model_Core_Session class is extending Mage_Core_Model_Session_Abstract, instead of Mage_Core_Model_Session, so the new methods in Mage_Core_Model_Session, included in the patches, are missing

Does Nexcessnet_Turpentine_Model_Core_Session extend Mage_Core_Model_Session_Abstract for any particular reason?

**Comment by miguelbalparda** [comment]
Which version of Turpentine are you using?

**Comment by viansolutions-com** [comment]
0.6.8

**Comment by miguelbalparda** [comment]
This seems to be a duplicated of https://github.com/nexcess/magento-turpentine/issues/1077
Have you checked that issue?

**Comment by viansolutions-com** [comment]
No, I didn't 
Deleting app/code/local/Mage/Core/Model/Session.php did the trick

Thanks

**Comment by beckindesigns** [comment]
Here is what worked for me and I don't see this answer anywhere.

In my case it was a turpentine setting. I used the backdoor to get into the admin area by going to the downloader page (yoursite.com/downloader) and logging in. After that, I changed **_System > Configurations > Turpentine > Varnish Options > General Options > Use VCL fix_** to "disabled".

"When Enable is selected, a VCL fix will be used to prevent formKey issues. If Disable is selected, an observer will be used."

It appears the observer did the trick. Also, I am not using an override of the `app/code/local/Mage/Core/Model/Session.php` file.
